use-cases/time-series-forecasting/walmart-sales/ML/requirements.txt

```
@@ -0,0 +1,11 @@
+matplotlib
```

```suggestion
matplotlib<3.5
```
For plotly compatibility